verify.c

```diff
@@ -345,8 +345,9 @@ static void log_verify_failure(struct verify_header *hdr, struct vcont *vc)
 
 	offset = vc->io_u->offset;
 	offset += vc->hdr_num * hdr->len;
-	log_err("%.8s: verify failed at file %s offset %llu, length %u\n",
-			vc->name, vc->io_u->file->file_name, offset, hdr->len);
+	log_err("%.8s: verify failed at file %s offset %llu, length %u, requested block(offset %llu, length %llu)\n",
```

Comment:
I'd re-wrap these lines at 79 characters (or thereabouts) and put the whole requested block message within the brackets...

Reply:
Thank you for your comments.  Changed